server/js/sockets.js

```diff
@@ -126,31 +125,34 @@ sockets.init = function socketInit(server) {
         for the topics
     */
     mqttClient.subscribe('status/#');
+    mqttClient.subscribe('camera/#');
```

Comment:
Two nits:
1) Can we move both of these to where the other subscription takes place (i.e. `271`) just so they're grouped together and easy to find
2) Can we use `'${Camera.base}/#'` instead of 'hardcoding' the stem/prefix of the topic

Reply:
Done!